pkg/k8s_utils/fake_k8s_client.go

```diff
@@ -53,21 +51,18 @@ func FakeCreateSecret(kc KubernetesClient, fakeAuthType, secretdatafilepath stri
 		return errors.New("undefined auth type")
 	}
 
-	secret.Namespace = kc.GetNameSpace()
+	secret.Namespace = kc.Namespace
 	data := make(map[string][]byte)
 
 	byteData, err := ioutil.ReadFile(secretdatafilepath)
 	if err != nil {
-		kc.logger.Error("Error reading secret data", zap.Error(err))
 		return err
 	}
 
 	data[dataname] = byteData
 	secret.Data = data
-	clientset := kc.clientset
-	_, err = clientset.CoreV1().Secrets("kube-system").Create(context.TODO(), secret, metav1.CreateOptions{})
+	_, err = kc.Clientset.CoreV1().Secrets("kube-system").Create(context.TODO(), secret, metav1.CreateOptions{})
```

Comment:
Should this be in a constant var?  => `kube-system`

Reply:
Will change this kc.Namespace